ipaserver/plugins/migration.py

```diff
@@ -28,7 +28,7 @@
 from ipalib import Command, Password, Str, Flag, StrEnum, DNParam, Bool
 from ipalib.cli import to_cli
 from ipalib.plugable import Registry
-from .user import NO_UPG_MAGIC
+from ipaserver.plugins.user import NO_UPG_MAGIC
```

Comment:
While @tiran did not think this change necessary, since this would be the only spot to have such a relative import and also by the "explicit is better than implicit" (maybe explicitly explicit in this case, I agree) I would keep it. It fixes `bad-python3-import` Pylint anyway and might be cleaner than just putting a pylint-disable here.